### PR TITLE
Add write access request form

### PR DESCRIPTION
**Pull request by timtebeek**
## What's changed?
Added write access [request form](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms), which will then show up in _each_ OpenRewrite repository, when people go to open a new issue.

## What's your motivation?
We want to document the way people can get involved with the project, especially for authors of library and frameworks we already cover.

## Anything in particular you'd like reviewers to focus on?
- Is it clear enough that we won't as easily grant write access to just anyone,
but specifically to people already involved with the library or framework covered in each module?
- Is it sufficiently clear from the comment at the bottom that we do expect some form of commitment to the project?
- Any other fields we might like to ask about?

## Have you considered any alternatives or workarounds?
We could also _only_ have this form in a dedicated repository for access requests, but that makes it harder to find when you come in through for instance `rewrite-hibernate`. 

## Any additional context
https://github.com/openrewrite/rewrite-docs/issues/188

### CODE REVIEW

**Comment by timtebeek** [comment]
A preview of the form is here: https://github.com/openrewrite/.github/blob/tim/write_access_request_form/.github/ISSUE_TEMPLATE/write_access_request.yml

**Review by rpau — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/write_access_request.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      description: What organization are you affiliated with? This helps us understand your relationship to the project. 
+      placeholder: ex. Acme Corp
+    validations:
+      required: true
+  - type: textarea
+    id: motivation
+    attributes:
+      label: Motivation
+      description: Why do you want write access to this repository?
+      placeholder: ex. I'm an active member of the community and would like to help triage issues.
```
Comment (mike-solomon) [inline .github/ISSUE_TEMPLATE/write_access_request.yml:29]:
```suggestion
      placeholder: e.g., I'm an active member of the community and would like to help triage issues.
```

**Review by mike-solomon — APPROVED** [review]
Two minor suggestions but other than that looks good to me. Thanks for doing this!

**Pull request merged**